JellyDust/UnitOfWork.cs

```diff
@@ -24,16 +24,26 @@ public IConnection Connection
             get
             {
                 VerifyNotDisposed();
+                if (_connection != null)
+                {
+                    _connection.VerifyNotDisposed();
+                }
+
                 return _connection ?? (_connection = new Connection(_connectionFactory));
             }
         }
 
-        public ITransaction Session
+        public ITransaction Transaction
```

Comment:
Breaking change?

Reply:
Yes. But easily fixed so I think its ok to just mention without increasing major

Reply:
Yup, probably just a one-line change in the IoC setup